public/add-to-storage.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+var addAnswersToSessionStorage = function(){
```

Comment:
Can you decide how you are bundling / wrapping up you js scripts ...
I really do not think having lots of different functions declared in the top level on many js files is a good way of doing things..